src/quo2/components/notifications/activity_log/view.cljs

```diff
@@ -155,18 +116,42 @@
     :style               style/unread-dot-container}
    [rn/view {:style style/unread-dot}]])
 
+(defmulti footer-item-view (fn [item _ _] (:type item)))
+
+(defmethod footer-item-view :button
+  [{:keys [label subtype disable-when] :as button} replying? reply-input]
+  (let [size         (if replying? 40 24)
+        common-style (when replying?
+                       {:padding-vertical 9
+                        :flex-grow        1
+                        :flex-basis       0})]
+    [button/button
+     (-> button
+         (assoc :size size)
+         (assoc :type subtype)
+         (assoc :disabled (and replying? (disable-when @reply-input)))
+         (update :style merge common-style {:margin-right 8}))
```

Comment:
@alwx is it by design that we override `styles` provided by client via `common-style {:margin-right 8}` or the order should be changed here?

Reply:
oh ok I suppose that's just an old code moved elsewhere

Reply:
Yes, that's true, it's an old code.